src/MicrosoftTeams.ts

```diff
@@ -551,12 +556,24 @@ namespace microsoftTeams {
             notifyFailure(reason?: string): void;
         }
 
+        export interface SaveResult {
```

Comment:
I don't think SaveResult is the right name for this; a result implies something that comes out of the save operation but actually this is passed in at the start of it. I think SaveParameters might be amore appropriate name for this.